### DISCUSSION TITLE
Correct definition for ETVH (16S18DA6VG) models

### DISCUSSION THREAD

**Discussion by MightContainNuts** [raomin/ESPAltherma#211] [General]
First and foremost - fantastic work - enjoy the beers.
The HAY (I do have a raspi4 tho) has been delayed somewhat, but the Daikin heatpump is up and running since 1 week.

Looking to see if anyone has found a better fit than the default.h  for ETVH models - ?

**Comment by TRIROG** [comment]
Bump, as I'm also interested.
